tests/api_tests.py

```diff
@@ -357,20 +357,18 @@ async def test_token_exchange_sad_path(test_app, api_client: APISessionClient):
 @pytest.mark.asyncio
 @pytest.mark.parametrize(
     "test_product_and_app",
-    _add_authorised_targets_to_request_params(
-        [
-            {
-                "scopes": ["urn:nhsd:apim:user-nhs-id:aal3:immunisation-history"],
-                "requested_proofing_level": "P9",
-                "identity_proofing_level": "P9",
-            },
-            {
-                "scopes": ["urn:nhsd:apim:user-nhs-id:aal3:immunisation-history"],
-                "requested_proofing_level": "P5",
-                "identity_proofing_level": "P9",
-            },
-        ]
-    ),
+    [
```

Comment:
Note that I removed this because it's not actually supposed to get as far as authorised targets anyway, so it just duplicates tests